client/go/outline/electron/main.go

```diff
@@ -41,6 +42,13 @@ const (
 	persistTun = true // Linux: persist the TUN interface after the last open file descriptor is closed.
 )
 
+// tun2socks exit codes. Must be kept in sync with definitions in "go_vpn_tunnel.ts"
+const (
+	exitCodeSuccess           = 0
```

Comment:
I propose we output a structured JSON in the stdout instead of the integers. Then we can forward a lot more info to the user.

Reply:
Same question as above, we are not showing anything when UDP is not supported currently, are we going to show anything to the user in the future?

Reply:
Ditto